### PR TITLE
Transfers bugfixes

### PR DESCRIPTION
**Pull request by easteregg**
## Issue 
https://app.clickup.com/t/34p59rd
https://app.clickup.com/t/3329km5
https://app.clickup.com/t/3pqjn4p
https://app.clickup.com/t/3u4b157
https://app.clickup.com/t/31evhex

## Description
- Fix some UX issue to properly update transfer input
- Fix wallet modal to be closed upon confirming an account
- Fix minValue requirements and error message passing from child/parent
- Update balance updater to (FORNOW) use 6 as decimals for USDT until RPC asset list is ready.
- Update asset mapping and allowed transfers to consider destination chain's supported asset.
- Enable KSM (Statemine => Picasso) and USDT (Picasso => Statemine) transfers.

### CODE REVIEW

File: frontend/apps/picasso/defi/polkadot/pallets/Assets/picasso.ts [hunk 1]
```diff
@@ -13,13 +13,16 @@ export async function picassoAssetsList(
...
-    return assetsList.map(asset => {
+    return assetsList.map((asset) => {
       return {
         name: asset.name.toUtf8(),
         id: new BigNumber(asset.id.toString()),
-        decimals: asset.decimals.toNumber(),
+        decimals:
+          asset.name.toUtf8().toUpperCase() !== "USDT"
+            ? asset.decimals.toNumber()
+            : 6, // TODO: Temporary assign 6 decimals to USDT until RPC is ready
```
Comment (andresmechali) [inline frontend/apps/picasso/defi/polkadot/pallets/Assets/picasso.ts:23]: You will manually update it to `6` then no?

File: frontend/packages/wallet/components/ConnectionModal.tsx [hunk 2]
```diff
@@ -106,7 +108,7 @@ export const ConnectionModal: React.FC<ConnectionModalProps> = ({
         key={wallet.walletId}
         onClick={(walletId: SupportedWalletId) => {
           onConnectPolkadotWallet(walletId)
-            .then((walletConnected) => {
+            .then((_walletConnected) => {
```
Comment (andresmechali) [inline frontend/packages/wallet/components/ConnectionModal.tsx:111]: Why not directly removing it?

**Review by andresmechali — APPROVED** [review]
LGTM

File: frontend/apps/picasso/defi/polkadot/pallets/Assets/picasso.ts (hunk 1 above)
Comment (easteregg) [reply frontend/apps/picasso/defi/polkadot/pallets/Assets/picasso.ts:23]: Yes, this is just for now until your PR is merged.

**Comment by easteregg** [comment]
@mergifyio refresh

**Pull request merged**